pkg/executor/build_test.go

```diff
@@ -522,6 +523,55 @@ func TestInitializeConfig(t *testing.T) {
 	}
 }
 
+func Test_newLayerCache(t *testing.T) {
+	tests := []struct {
+		description string
+		opts        *config.KanikoOptions
+		assert      func(layerCache cache.LayerCache) error
```

Comment:
Not a huge deal, but this style effectively undoes the point of table-driven tests. I'd consider just writing three `TestNewLayerCache_Foo` methods, rather than squeezing them into a table-driven test with three different checks.

I won't block the PR on it, just something to consider.

Reply:
Updated in c55854f7